doc/release_notes/release_notes_2.3.rst

```diff
@@ -0,0 +1,133 @@
+source code is maintained in the
+https://github.com/projectacrn/acrn-hypervisor repository and includes
+folders for the ACRN hypervisor, the ACRN device model, tools, and
+documentation. You can either download this source code as a zip or
+tar.gz file (see the `ACRN v2.3 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v2.3>`_) or
+use Git clone and checkout commands::
+
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   cd acrn-hypervisor
+   git checkout v2.3
+
+The project's online technical documentation is also tagged to
+correspond with a specific release: generated v2.3 documents can be
+found at https://projectacrn.github.io/2.3/.  Documentation for the
+latest under-development branch is found at
+https://projectacrn.github.io/latest/.
+
+ACRN v2.3 requires Ubuntu 18.04.  Follow the instructions in the
+:ref:`rt_industry_ubuntu_setup` to get started with ACRN.
+
+
+What's New in v2.3
+******************
+
+Enhanced GPU passthru (GVT-d)
+  GPU passthru (GVT-d) to Windows as a guest is now enabled for 11th Gen
+  Intel® Core™ processors (codenamed Tiger Lake-UP3).
+
+Shared memory based inter-VM communication (ivshmem) is extended
+  ivshmem now supports interrupts (See :ref:`ivshmem-hld`).
+
+Enhanced vUART support
+  Added console support using legacy vUART (0x3F8-like) and
+  added PCI vUART (up to 8) for VM-to-VM communication.
+
+End-to-end secure boot improvement
+  OVMF can be loaded now as two blobs, one for code and the other for data.
+  The code blob can be verified by the Service VM's ``dm-verify`` as
+  a step in the end-to-end secure boot.
+
+Enhanced system shutdown
+  The pre-launched VM may now initiate a system shutdown or reset.
+
+Removed deprivileged boot mode support
+  ACRN has supported deprivileged boot mode to ease the integration of
+  Linux distributions such as Clear Linux. Unfortunately, deprivileged boot
+  mode limits ACRN's scalability and is unsuitable for ACRN's hybrid
+  hypervisor mode. In ACRN v2.2, deprivileged boot mode was no longer the default
+  and completely removed in ACRN v2.3. We're focusing instead
+  on using the simpler and more scalable multiboot2 boot (via Grub).
+  Multiboot2 is not supported in
+  Clear Linux so we have chosen Ubuntu (and Yocto Project) as the
+  preferred Service VM OSs moving forward.
+
+Document updates
+****************
+
+New and updated reference documents are available, including:
+
+.. rst-class:: rst-columns2
+
+* :ref:`GVT-g-porting`
+* :ref:`vbsk-overhead`
+* :ref:`asm_coding_guidelines`
+* :ref:`c_coding_guidelines`
+* :ref:`contribute_guidelines`
+* :ref:`doc_guidelines`
+* :ref:`hld-devicemodel`
+* :ref:`hld-overview`
+* :ref:`hld-power-management`
+* :ref:`hld-security`
+* :ref:`hld-trace-log`
+* :ref:`hld-virtio-devices`
+* :ref:`ivshmem-hld`
+* :ref:`l1tf`
+* :ref:`modularity`
+* :ref:`sw_design_guidelines`
+* :ref:`rt_industry_ubuntu_setup`
+* :ref:`introduction`
+* :ref:`release_notes_2.2`
```

Comment:
It's a little odd to call out that we updated the Release Note for v2.2 as part of our v2.3 release.